### PR TITLE
[DO-NOT-MERGE] Screen capturer

### PR DESCRIPTION
**Pull request by Lucashuang0802**
I have tried it, it doesn't seem to be working.

### CODE REVIEW

**Comment by Lucashuang0802** [comment]
In associated with: https://github.com/opentok/opentok-ios-sdk-samples/pull/143

**Comment by Lucashuang0802** [comment]
BTW, the device I was trying on was iPhone 7 Plus simulator.

**Comment by Lucashuang0802** [comment]
#9

File: 4.Screen-Sharing/4.Screen-Sharing/ScreenCapturer.swift [hunk 1]
```diff
@@ -37,14 +37,14 @@ class ScreenCapturer: NSObject, OTVideoCapture {
     fileprivate func resizeAndPad(image img: UIImage) -> CGImage {
         let source = img.cgImage!
         let size = CGSize(width: source.width, height: source.height)
-        let destSizes = dimensions(forInputSize: size)
+//        let destSizes = dimensions(forInputSize: size)
```
Comment (robjperez) [inline 4.Screen-Sharing/4.Screen-Sharing/ScreenCapturer.swift]: Please remove completely this line instead of commenting it

**Comment by Lucashuang0802** [comment]
@robjperez updated, but it doesn't seem to be working. This PR is just a reminder here.

**Pull request closed without merging**